deploy/cr.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: "logging.openshift.io/v1alpha1"
 kind: "Elasticsearch"
 metadata:
-  name: "elastic1"
+  name: "elasticsearch"
 spec:
   nodeSpec:
     image: quay.io/openshift/logging-elasticsearch5:latest
```

Comment:
For some reason this image is private not public - needs credentials to pull - change to be consistent with cluster-logging-operator: `image: openshift/origin-logging-elasticsearch5:latest`